README.md

```diff
@@ -127,6 +127,41 @@ Two simple steps to override require in your tests:
     - therefore specify it exactly as in the require statement inside the tested file
     - values themselves are key/value pairs of functions/properties and the appropriate override
 
+## Raise error if proxying a modulePath that the module under test does not reference
+
+By default `proxyquire` will stub out modulePaths, whatever their path may be.
```

Comment:
Avoid "stub out," it's unclear what that means. It's also not true, as the sentence is written.